--- a/setup-env.orig
+++ b/setup-env
@@ -98,8 +98,9 @@
     `find ${ROOTDIR}/sources/meta-freescale*/conf/machine \
         ${ROOTDIR}/sources/meta-fsl*/*/*/conf/machine \
 	${ROOTDIR}/sources/meta-qoriq/conf/machine \
+	${ROOTDIR}/sources/meta-scalys/conf/machine \
         -name *.conf 2>/dev/null \
-        |egrep "/(mpc|b4|bsc|c29|p1|p2|p3|p4|p5|t1|t2|t4|ls|lx).*conf$" |sort \
+        |egrep "/(mpc|b4|bsc|c29|p1|p2|p3|p4|p5|t1|t2|t4|ls|lx|trustsom).*conf$" |sort \
         |xargs echo |sed -e 's,[^ ]*/,,g;s,.conf,,g'`
 "
 
@@ -159,12 +160,14 @@
     valid_num=`find ${ROOTDIR}/sources/meta-freescale*/conf/machine \
         ${ROOTDIR}/sources/meta-fsl*/*/*/conf/machine \
 	${ROOTDIR}/sources/meta-qoriq/conf/machine \
+	${ROOTDIR}/sources/meta-scalys/conf/machine \
         -name ${MACHINE}.conf 2>/dev/null |wc -l`
     if [ "1" -lt "$valid_num" ];then
         echo "ERROR: possible error may occur due to duplicate ${MACHINE}.conf exist:"
         find ${ROOTDIR}/sources/meta-freescale*/conf/machine \
             ${ROOTDIR}/sources/meta-fsl*/*/*/conf/machine \
-	    ${ROOTDIR}/sources/meta-qoriq/conf/machine \ 
+	    ${ROOTDIR}/sources/meta-qoriq/conf/machine \
+	    ${ROOTDIR}/sources/meta-scalys/conf/machine \
             -name ${MACHINE}.conf 2>/dev/null
         echo "Please remove the useless layer under ${ROOTDIR}/sources"
         clean_up && return
@@ -197,6 +200,7 @@
     meta-qoriq \
     meta-imx/meta-bsp \
     meta-imx/meta-ml \
+    meta-scalys \
 "
 EULA_FILE="$FSLROOTDIR/EULA"
 
